[Bug] ternary alignes weirdly in JSX

**Describe the bug**

The ternary code in `JSX` aligns weirdly. 

**Context**

_It is best to provice which config (e.g. base or react) and rule is affected._
- Rule: _ multiline-ternary_
- config: ?

 15:10  error  Expected newline between test and consequent of ternary expression  multiline-ternary
 15:18  error  Expected newline between consequent and alternate of ternary expression  multiline-ternary


**To Reproduce**

Steps to reproduce the behavior:
Just look at the screenshots.

**Expected behavior**
![Bildschirmfoto 2020-10-09 um 14 17 14](https://user-images.githubusercontent.com/18395108/95582309-be2e0500-0a3a-11eb-8df7-9f18c5ad52c8.png)
The ternary should be aligned better so it's clear that it is a ternary.

**Screenshots**
![Bildschirmfoto 2020-10-09 um 14 21 16](https://user-images.githubusercontent.com/18395108/95582283-b79f8d80-0a3a-11eb-94c7-137d993f12e3.png)

--- a/base.js
+++ b/base.js
@@ -225,7 +225,7 @@ module.exports = {
     'max-params': ['warn', { max: 3 }], // https://eslint.org/docs/rules/max-params
     'max-statements': 'off', // https://eslint.org/docs/rules/max-statements
     'max-statements-per-line': ['error', { max: 1 }], // https://eslint.org/docs/rules/max-statements-per-line
-    'multiline-ternary': ['error', 'always-multiline'], // https://eslint.org/docs/rules/multiline-ternary
+    'multiline-ternary': ['off', 'always-multiline'], // https://eslint.org/docs/rules/multiline-ternary
     'new-cap': 'off', // this is handled by babel/new-cap
     'new-parens': ['error'], // https://eslint.org/docs/rules/new-parens
     'newline-per-chained-call': ['error', { ignoreChainWithDepth: 2 }], // https://eslint.org/docs/rules/newline-per-chained-call

